app/main.py

```diff
@@ -158,13 +165,29 @@
     'CACHE_DEFAULT_TIMEOUT': 60
 })
 
+# ----------------------------------------------------------------------------------------------------------------------
+# Diverse
+# The following block is dedicated to the definition of variables and functions that will be used in main
+
+
+def is_hour_between(sunrise, sunset, alert_time):
+
+    alert_time = datetime.fromisoformat(str(alert_time)) + timedelta(hours=2)
+    alert_time = alert_time.time()
+
+    is_between = False
+    is_between |= sunrise <= alert_time <= sunset
+    is_between |= sunset <= sunrise and (sunrise <= alert_time or alert_time <= sunset)
```

Comment:
Could you please explain in what case `sunset <= sunrise` evaluates to `True`?

Reply:
In a rare case of working with UTC let's say in Hawai or Japan it may occur that sunset <= sunrise turns to be True. Agree that won't never be the case for us but leaving it there

Reply:
Well noted, thanks for the indication!